lumicks/pylake/scan.py

```diff
@@ -91,10 +91,137 @@ def _plot(self, image, frame=1, **kwargs):
             aspect=(image.shape[0] / image.shape[1]) * (x_um / y_um)
         )
 
-        plt.imshow(image, **{**default_kwargs, **kwargs})
+        if not image_handle:
+            image_handle = plt.imshow(image, **{**default_kwargs, **kwargs})
+        else:
+            # Updating the image data in an existing plot is a lot faster than re-plotting with `imshow`.
+            image_handle.set_data(image)
```

Comment:
It would be good to make this an argument docstring instead of a comment. It would explain why `image_handle` is even needed as an argument.

Reply:
Fair enough. I've added a docstring to the function.

```python
        """Plot a scan frame.

        Parameters
        ----------
        image : array_like
            Image data
        frame : int
            Frame index (starting at 1).
        image_handle : `matplotlib.image.AxesImage` or None
            Optional image handle which is used to update plots with new data rather than reconstruct them (better for
            performance).
        **kwargs
            Forwarded to :func:`matplotlib.pyplot.imshow`
        """
```

Reply:
I think that you can completely remove the comment now. It's clear from the docstring.